get_resilabels.py

```diff
@@ -1,186 +1,170 @@
+
+        # convert the residue name for each token in each line
+        # for example, the following line: ["A:LEU  75", "A:LEU  65"] -> ["A:LEU:75", "A:LEU:65"]
+        line = [
+            token[:5] + ":" + str(int(token[5:])) if len(token) > 0 else token
+            for token in line
+        ]
+        reformatted_alignment_lines.append(line)
+
+    return reformatted_alignment_lines, proteins
 
 
 def main(argv=None):
-    parser = argparse.ArgumentParser(description=__doc__)
-    parser.add_argument('--ssm_filename',
-                        required=True,
-                        help="Path to a .dat output file generated by SSM's multiple sequence alignment tool")
-    parser.add_argument('--output_dir',
-                        required=False,
-                        default='./',
-                        help="Directory where label files will be written")
-    parser.add_argument('--include_loconf',
-                        type=bool,
-                        required=False,
-                        default=False,
-                        help="Include residues aligned with low confidence in label files (default: False)")
-    args = parser.parse_args(argv)
+    # Set up command line arguments
+    import argparse
+
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "--input_gesamt",
+        required=True,
+        type=str,
+        help="A .txt file produced by redirecting output from gesamt to a file",
+    )
+    parser.add_argument(
+        "--output_path", required=True, type=str, help="Path to output directory"
+    )
+    parser.add_argument(
+        "--proteins",
+        required=False,
+        type=str,
+        nargs="+",
+        help='Optionally, provide a space-delimited list of names of all proteins in the gesamt input file. Protein names should be specified in the order that the proteins were inputted to the gesamt program. For example, if you called gesamt with "gesamt foo1.pdb foo2.pdb foo3.pdb", you can specify "--proteins foo1 foo2 foo3"',
+    )
 
-    input_filelines = get_filelines(args.ssm_filename)
-    pdbs = extract_pdbs(input_filelines)
-    hiconf_alignments, loconf_alignments = extract_alignments(input_filelines, pdbs)
-    alignments = loconf_alignments + hiconf_alignments if args.include_loconf else hiconf_alignments
-    output_labelfiles(args.output_dir, pdbs, alignments)
+    args = parser.parse_args(argv)
+    input_gesamt = args.input_gesamt
+    output_path = args.output_path
+    input_proteins = args.proteins
+
+    # input_gesamt = "two_queries/two_queries.txt"
+    with open(input_gesamt, "r") as r_open:
+        gesamt_lines = [
+            line.strip() for line in r_open.readlines() if len(line.strip()) > 0
+        ]
+
+    # parse the input file differently depending on whether there are 2 queries or >2 queries
+    if "reading QUERY structure" in "".join(gesamt_lines) or "reading FIXED structure" in "".join(gesamt_lines):
```

Comment:
This join is fairly ineffective way of enable searching, but doing it twice is twice as bad. 
```python
gesamt_text = "".join(gesamt_lines)
if "reading QUERY structure" in gesamt_text or ...:
```